### DISCUSSION TITLE
Why do websocket disconnects happen?

### DISCUSSION THREAD

**Discussion by khaimong** [sebr/bhyve-home-assistant#264] [General]
Just curious why I get websocket disconnects sometimes.  Lately, I have been seeing more, five or more a day. It used to be once a week or less.  My Internet connection seems good, because the websocket reconnect is fast.

**Comment by sebr** [comment]
Networks are inherently fragile. It could be a disruption anywhere between your end and the BHyve API servers. You will notice issues with Websockets significantly more than other requests because of their keep-alive nature. YMMV. ¯\_(ツ)_/¯
